xcache/image-config.d/00-functions.sh

```diff
@@ -0,0 +1,25 @@
+supervisord_is_enabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf ]]
+}
+
+
+supervisord_is_disabled () {
+    local service="$1"
+    [[ -e /etc/supervisord.d/${service}.conf.disabled ]]
+}
```

Comment:
There is a small part of me that notices that there is a slight difference between `supervisord_is_disabled service` and `! supervisord_is_enabled service`.  Do you ever care about the case where the .conf/.disabled file is missing  (and thus both functions report false) ?

Anyway, fine how you have it.  Maybe it's nice for documentation that it kind of spells out what the .disabled extension means.  But otherwise yeah if the state is meant to be boolean, I guess `supervisord_is_disabled` could be written `! supervisord_is_enabled`.  (If you were so inclined, that is.)

Reply:
I use these functions to test if the service _can_ be enabled/disabled. If both files are missing then it can neither be enabled nor disabled. So I think the logic is fine.